File: scripts/update_homepage.py
import os
import re

def update_file(path, replacements):
    with open(path, 'r', encoding='utf-8') as f:
        content = f.read()
    
    for old, new in replacements:
        if old in content:
            content = content.replace(old, new)
        else:
            print(f"Warning: Could not find '{old[:50]}...' in {path}")
            
    with open(path, 'w', encoding='utf-8') as f:
        f.write(content)

# Update pt/index.html
pt_index_replacements = [
    (
        '<h2 class="font-display font-bold text-dark leading-[1.08] tracking-tight animate-fade-in-up delay-100" style="font-size:clamp(2.2rem, 5.5vw, 5rem);">\n                    Recupere Até 70% do Tempo da Sua Equipa <span class="text-accent">Com IA e Automação</span>\n                </h2>',
        '<h2 class="font-display font-bold text-dark leading-[1.08] tracking-tight animate-fade-in-up delay-100" style="font-size:clamp(2.2rem, 5.5vw, 5rem);">\n                    Sistemas de IA Personalizados para Empresas\n                </h2>'
    ),
    (
        '<p class="max-w-xl mt-6 text-base md:text-lg text-slate-600 leading-relaxed animate-fade-in-up delay-200">\n                    Criamos agentes de IA, automatizamos processos e devolvemos horas à sua equipa — para decisões que fazem a diferença.\n                </p>',
        '<p class="max-w-xl mt-6 text-base md:text-lg text-slate-600 leading-relaxed animate-fade-in-up delay-200">\n                    Analisamos os seus processos, desenvolvemos agentes, automações e ferramentas personalizadas para reduzir custos, recuperar tempo e acelerar o crescimento.\n                </p>'
    ),
    (
        '<span class="hidden sm:inline">Diagnóstico gratuito — Ver quanto posso poupar</span>',
        '<span class="hidden sm:inline">Descobrir oportunidades no meu negócio</span>'
    ),
    (
        '<a href="#problema" class="inline-flex items-center justify-center gap-2 px-6 py-3.5 rounded-full text-sm font-medium transition-all duration-300 bg-white border border-slate-200 text-slate-700 hover:bg-slate-50 hover:border-slate-300 shadow-sm">\n                            Descobrir mais\n                            <svg width="16" height="16" viewBox="0 0 16 16" fill="none"><path d="M8 3L8 13M8 13L4 9M8 13L12 9" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"></path></svg>\n                        </a>',
        '<a href="casos-estudo.html" class="inline-flex items-center justify-center gap-2 px-6 py-3.5 rounded-full text-sm font-medium transition-all duration-300 bg-white border border-slate-200 text-slate-700 hover:bg-slate-50 hover:border-slate-300 shadow-sm">\n                            Ver casos reais\n                            <svg width="16" height="16" viewBox="0 0 16 16" fill="none"><path d="M8 3L8 13M8 13L4 9M8 13L12 9" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"></path></svg>\n                        </a>'
    ),
    (
        '<a href="chatbot.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Chatbot</a>\n                            <a href="emailbot.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Emailbot</a>\n                            <a href="qualificar-leads-crm.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Qualificar Leads CRM</a>\n                            <a href="agendar-reunioes.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agendar Reuniões</a>\n                            <a href="agentes-de-voz.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agentes de Voz</a>\n                            <a href="agentes-de-texto.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agentes de Texto</a>\n                            <a href="redes-sociais.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Redes Sociais</a>',
        '<a href="agentes-conversacao.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agentes de Conversação</a>\n                            <a href="emailbot.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Emailbot</a>\n                            <a href="qualificar-leads-crm.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Qualificar Leads CRM</a>\n                            <a href="agendar-reunioes.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agendar Reuniões</a>\n                            <a href="agentes-de-voz.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Agentes de Voz</a>\n                            <a href="ferramentas-personalizadas.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Ferramentas Personalizadas</a>\n                            <a href="redes-sociais.html" class="px-4 py-2.5 rounded-xl hover:bg-blue-50 text-xs text-slate-600 hover:text-accent font-medium transition-colors">Redes Sociais</a>'
    ),
    (
        '<a href="chatbot.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Chatbot</a>\n                    <a href="emailbot.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Emailbot</a>\n                    <a href="qualificar-leads-crm.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Qualificar CRM</a>\n                    <a href="agendar-reunioes.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agendar Reuniões</a>\n                    <a href="agentes-de-voz.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agentes de Voz</a>\n                    <a href="agentes-de-texto.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agentes de Texto</a>\n                    <a href="redes-sociais.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Redes Sociais</a>',
        '<a href="agentes-conversacao.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agentes de Conversação</a>\n                    <a href="emailbot.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Emailbot</a>\n                    <a href="qualificar-leads-crm.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Qualificar CRM</a>\n                    <a href="agendar-reunioes.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agendar Reuniões</a>\n                    <a href="agentes-de-voz.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Agentes de Voz</a>\n                    <a href="ferramentas-personalizadas.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Ferramentas Personalizadas</a>\n                    <a href="redes-sociais.html" class="mobile-link text-lg font-medium text-slate-700 hover:text-accent">Redes Sociais</a>'
    )
]

update_file('/Users/joseteixeira/Desktop/Antigravity Stuf/Mobizze2.0/pt/index.html', pt_index_replacements)

# Add disclaimer after stats bar in pt/index.html
with open('/Users/joseteixeira/Desktop/Antigravity Stuf/Mobizze2.0/pt/index.html', 'r', encoding='utf-8') as f:
    pt_content = f.read()

disclaimer_pt = '            </div>\n            <div class="max-w-5xl mx-auto mt-6 text-center">\n                <p class="text-xs text-slate-400">Dados agregados de projetos implementados entre 2024 e 2026. Resultados variam consoante os processos, volume e sistemas de cada empresa.</p>\n            </div>'
if 'Dados agregados de projetos' not in pt_content:
    pt_content = pt_content.replace('            </div>\n        </div>\n\n        <!-- Clientes e Parceiros -->', disclaimer_pt + '\n        </div>\n\n        <!-- Clientes e Parceiros -->')
    with open('/Users/joseteixeira/Desktop/Antigravity Stuf/Mobizze2.0/pt/index.html', 'w', encoding='utf-8') as f:
        f.write(pt_content)
        
print("Updated pt/index.html")

File: scripts/update_homepage.js
const fs = require('fs');
const path = require('path');

const ptPath = path.join(__dirname, '../pt/index.html');
const enPath = path.join(__dirname, '../en/index.html');

const ptContent = fs.readFileSync(ptPath, 'utf8');
const enContent = fs.readFileSync(enPath, 'utf8');

const ptNewSection = `
<section class="relative bg-slate-50 py-24 overflow-hidden border-t border-slate-100">
    <div class="max-w-5xl mx-auto px-6">
        <div class="flex flex-col md:flex-row justify-between items-end gap-6 mb-12 reveal">
            <div>
                <p class="text-[10px] md:text-xs font-mono tracking-[0.25em] uppercase mb-3 text-accent">Impacto Comprovado</p>
                <h3 class="text-2xl md:text-3xl font-display font-bold text-dark tracking-tight">Casos de Sucesso B2B</h3>
            </div>
            <a href="casos-estudo.html" class="inline-flex items-center gap-2 px-5 py-2.5 rounded-full text-xs font-medium bg-white border border-slate-200 text-slate-700 hover:bg-blue-50 hover:text-accent hover:border-blue-200 transition-all shadow-sm">
                Ver todos os casos
                <svg width="14" height="14" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h14"></path><path d="m12 5 7 7-7 7"></path></svg>
            </a>
        </div>
        
        <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
            <div class="bg-white p-8 rounded-3xl border border-slate-200 shadow-sm reveal delay-100">
                <div class="inline-block px-3 py-1 rounded-full bg-blue-50 text-accent text-[10px] font-bold uppercase tracking-wider mb-4">Portugal Textile</div>
                <h4 class="text-xl font-display font-bold text-dark mb-3">Automação Comercial</h4>
                <p class="text-slate-600 text-sm mb-6 leading-relaxed">Implementámos IA para triar centenas de emails técnicos de encomendas, reduzindo em 80% o trabalho administrativo e aumentando a velocidade de resposta em 5x.</p>
                <div class="pt-4 border-t border-slate-100">
                    <p class="text-sm italic text-slate-700 mb-2">"O impacto foi imediato. A equipa comercial deixou de perder horas a ler emails técnicos e passou a focar-se apenas em fechar negócio."</p>
                    <p class="text-[10px] font-bold text-dark uppercase tracking-wide">— Diretor Comercial</p>
                </div>
            </div>
            
            <div class="bg-white p-8 rounded-3xl border border-slate-200 shadow-sm reveal delay-200">
                <div class="inline-block px-3 py-1 rounded-full bg-blue-50 text-accent text-[10px] font-bold uppercase tracking-wider mb-4">Portugal Shoes</div>
                <h4 class="text-xl font-display font-bold text-dark mb-3">Sincronização Logística</h4>
                <p class="text-slate-600 text-sm mb-6 leading-relaxed">Assistente IA que cruza dados de fornecedores com planeamento interno, reduzindo os atrasos de produção em 40% com rastreio automático a 100%.</p>
                <div class="pt-4 border-t border-slate-100">
                    <p class="text-sm italic text-slate-700 mb-2">"O melhor investimento tecnológico que fizemos. Agora, a IA prevê as falhas de matéria-prima antes mesmo delas acontecerem."</p>
                    <p class="text-[10px] font-bold text-dark uppercase tracking-wide">— Gestor de Operações</p>
                </div>
            </div>
        </div>
    </div>
</section>
`;

const enNewSection = `
<section class="relative bg-slate-50 py-24 overflow-hidden border-t border-slate-100">
    <div class="max-w-5xl mx-auto px-6">
        <div class="flex flex-col md:flex-row justify-between items-end gap-6 mb-12 reveal">
            <div>
                <p class="text-[10px] md:text-xs font-mono tracking-[0.25em] uppercase mb-3 text-accent">Proven Impact</p>
                <h3 class="text-2xl md:text-3xl font-display font-bold text-dark tracking-tight">B2B Success Stories</h3>
            </div>
            <a href="case-studies.html" class="inline-flex items-center gap-2 px-5 py-2.5 rounded-full text-xs font-medium bg-white border border-slate-200 text-slate-700 hover:bg-blue-50 hover:text-accent hover:border-blue-200 transition-all shadow-sm">
                View all cases
                <svg width="14" height="14" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h14"></path><path d="m12 5 7 7-7 7"></path></svg>
            </a>
        </div>
        
        <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
            <div class="bg-white p-8 rounded-3xl border border-slate-200 shadow-sm reveal delay-100">
                <div class="inline-block px-3 py-1 rounded-full bg-blue-50 text-accent text-[10px] font-bold uppercase tracking-wider mb-4">Portugal Textile</div>
                <h4 class="text-xl font-display font-bold text-dark mb-3">Commercial Automation</h4>
                <p class="text-slate-600 text-sm mb-6 leading-relaxed">We implemented AI to sort hundreds of technical order emails, reducing admin work by 80% and increasing response speed by 5x.</p>
                <div class="pt-4 border-t border-slate-100">
                    <p class="text-sm italic text-slate-700 mb-2">"The impact was immediate. The team stopped wasting hours reading technical emails and focused solely on closing deals."</p>
                    <p class="text-[10px] font-bold text-dark uppercase tracking-wide">— Commercial Director</p>
                </div>
            </div>
            
            <div class="bg-white p-8 rounded-3xl border border-slate-200 shadow-sm reveal delay-200">
                <div class="inline-block px-3 py-1 rounded-full bg-blue-50 text-accent text-[10px] font-bold uppercase tracking-wider mb-4">Portugal Shoes</div>
                <h4 class="text-xl font-display font-bold text-dark mb-3">Logistics Sync</h4>
                <p class="text-slate-600 text-sm mb-6 leading-relaxed">AI assistant that cross-references supplier data with internal planning, reducing production delays by 40% with 100% automated tracking.</p>
                <div class="pt-4 border-t border-slate-100">
                    <p class="text-sm italic text-slate-700 mb-2">"The best technological investment we've made. Now, the AI predicts raw material shortages before they even happen."</p>
                    <p class="text-[10px] font-bold text-dark uppercase tracking-wide">— Operations Manager</p>
                </div>
            </div>
        </div>
    </div>
</section>
`;

function replaceSection(content, newSection) {
    const startIndex = content.indexOf('<section class="relative bg-dark py-16 overflow-hidden">');
    if (startIndex === -1) return content;
    
    // Find the closing tag of this section
    let depth = 0;
    let i = startIndex;
    let endIndex = -1;
    
    while (i < content.length) {
        if (content.substr(i, 8) === '<section') depth++;
        if (content.substr(i, 9) === '</section>') {
            depth--;
            if (depth === 0) {
                endIndex = i + 10;
                break;
            }
        }
        i++;
    }
    
    if (endIndex === -1) return content;
    
    return content.substring(0, startIndex) + newSection + content.substring(endIndex);
}

fs.writeFileSync(ptPath, replaceSection(ptContent, ptNewSection), 'utf8');
fs.writeFileSync(enPath, replaceSection(enContent, enNewSection), 'utf8');

console.log('Homepage layout updated for PT and EN');
